package.json

```diff
@@ -48,6 +48,7 @@
     "creditcards": "2.1.2",
     "cross-env": "4.0.0",
     "debug": "2.2.0",
+    "diff": "1.4.0",
```

Comment:
Since this is not depending on #13367 anymore, this can be merged on its own, I've updated the shrinkmap :)